src/alert-manager/deploy/alert-configmap.yaml.template

```diff
@@ -18,12 +18,13 @@ data:
       receiver: pai-alert
       group_wait: 30s
       group_interval: 5m
-      repeat_interval: 4h
+      repeat_interval: {{ cluster_cfg["alert-manager"]["repeat-interval"] }}
       group_by: [alertname]
     receivers:
     - name: "pai-alert"
       email_configs:
         - to: {{ alert_info["receiver"] }}
+          send_resolved: true
           html: '{{ "{{" }} template "email.pai.html" . {{ "}}" }}'
```

Comment:
Let's make sure resolve mail have a green title.

Reply:
I do not know how to pass this into template to generate a different email.